### PR TITLE
Portfolio: desktop version

### PR DESCRIPTION
**Pull request by ShaliniNayan**
For the fourth project in my portfolio website:
- Used media queries to website displays correctly on desktop screens.
- Make sure there are no linter errors
- Make sure to use the correct GitHub flow
- Make sure documented work in a professional way

- Project requirements:
1.  Make sure the desktop version looks like the one in the Figma design template that chose. 
2. Figma design templates include two input fields (for first name and last name) in the desktop version and one input field (for full name) in the mobile version 
3. Use media queries in a CSS file to make the desktop version look like the template selected (Desktop: >=768px).
4. In order to lay out the elements on the page should use Grid and Flexbox.
6. Try to replicate the design as much as possible (e.g., font, colors, images, text, margins)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,230 +2,441 @@
...
+            <li class="arrows">
+              <img src="images/arrow-down.png" alt="arrow.png" />
+            </li>
+          </ul>
+        </div>
+        <ul class="icons-lang">
+          <li class="lang-icons"><img src="images/js.png" alt="js" />Javascript</li>
+          <li class="lang-icons"><img src="images/html.png" alt="html" />HTML</li>
+          <li class="lang-icons"><img src="images/css.png" alt="Ellipse" />CSS</li>
+        </ul>
```
Comment (didierganthier) [inline index.html:360]: Good job here 💯 but make sure to adjust the position of the text and the image,  the text is supposed to be below, this is what yours looks like 👇🏽 

Screenshot 2023-03-31 at 00 59 06

And this is what the Figma looks like 👇🏽 

Screenshot 2023-03-31 at 01 00 45

File: index.html [hunk 2]
```diff
@@ -2,230 +2,441 @@
...
+            <li>
+              
+            </li>
+            <li>
+              <img src="images/angellist.png" alt="angellist"
+              />
+            </li>
+          </ul>
+        </header>
+      </section>
```
Comment (didierganthier) [inline index.html:91]: I love your design 👍🏽 but it lacks alignment, the elements would look better if they were vertically aligned, this is what you have so far 👇🏽 

Screenshot 2023-03-31 at 01 03 49

**Review by didierganthier — CHANGES_REQUESTED** [review]
Hi @ShaliniNayan ,

🎉 Good job so far! You're making great progress on this project, but there are still a few areas that need attention before we can move on to the next step.



## Highlights
- Descriptive PR ✔️ 
- Linters passing ✔️ 
- Correct GitHub flow ✔️ 

## Required Changes 🛠
- [ ] Nice overall 💯  but make sure to also adjust the font weights 🏋🏽  to match the design, the better it looks the better it feels 😉 

- Please review the comments under the review for specific changes that still need to be made.

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code.

🚀 Keep pushing forward, you're almost there! If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Cheers and happy coding! 👏👏👏

File: index.html [hunk 3]
```diff
@@ -2,230 +2,441 @@
...
+            </p>
+          </article>
+          <ul class="skills">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          
         </div>
       </div>
```
[the remark this answers is not in the document]
Comment (ShaliniNayan) [reply index.html]: Noted and thank you for your kind suggestion.

File: index.html (hunk 1 above)
Comment (ShaliniNayan) [reply index.html:363]: Noted and thank you for your kind suggestion.

File: index.html (hunk 2 above)
Comment (ShaliniNayan) [reply index.html]: Noted and thank you for your kind suggestion.

File: index.html [hunk 4]
```diff
@@ -2,230 +2,447 @@
...
+              <img src="images/angellist.png" alt="angellist"
+              />
+            </li>
+           
+          </ul>
+        </header>
+        
+      </section>
+      
+       <section class="projects-section" id="portfolio">
```
[the remark this answers is not in the document]
Comment (ShaliniNayan) [reply index.html:95]: Noted Thank you. Try to make the change.

**Review by Favourezeugwa — APPROVED** [review]
## STATUS: APPROVED 🥇 🏆 🍹 

Hi @ShaliniNayan ,

I am Favour 👩🏽 , your reviewer for this project milestone.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Yayyy! 💃🏽, Congratulations! 🎉 🎉🎉

happy

## Highlights 💯 
- No Linter errors ✔️
- Good commit messages ✔️
- Professional Readme file ✔️
- Descriptive PR ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ShaliniNayan** [comment]
Thank You....

**Pull request merged**